.gitignore

```diff
@@ -8,3 +8,4 @@ node_modules
 
 # For gh-pages
 .ghpages
+docs/*
```

Comment:
would this make sense in ``build/docs`` ? That way it would be automatically ignored like other artifacts.